cmd/kubectl-package/deps/command.go

```diff
@@ -0,0 +1,127 @@
+package deps
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/dig"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"package-operator.run/package-operator/cmd/kubectl-package/buildcmd"
+	"package-operator.run/package-operator/cmd/kubectl-package/rootcmd"
+	"package-operator.run/package-operator/cmd/kubectl-package/treecmd"
+	"package-operator.run/package-operator/cmd/kubectl-package/updatecmd"
+	"package-operator.run/package-operator/cmd/kubectl-package/validatecmd"
+	"package-operator.run/package-operator/cmd/kubectl-package/versioncmd"
+	internalcmd "package-operator.run/package-operator/internal/cmd"
+)
+
+func ProvideIOStreams() rootcmd.IOStreams {
+	return rootcmd.IOStreams{
+		In:     os.Stdin,
+		Out:    os.Stdout,
+		ErrOut: os.Stderr,
+	}
+}
+
+func ProvideArgs() []string {
+	return os.Args[1:]
+}
+
+type RootSubCommandResult struct {
+	dig.Out
+
+	SubCommand *cobra.Command `group:"rootSubCommands"`
+}
+
+func ProvideTreeCmd(rendererFactory treecmd.RendererFactory) RootSubCommandResult {
+	return RootSubCommandResult{
+		SubCommand: treecmd.NewCmd(
+			rendererFactory,
+		),
+	}
+}
+
+func ProvideRendererFactory(scheme *runtime.Scheme, f LogFactory) treecmd.RendererFactory {
+	return &defaultRendererFactory{
+		logFactory: f,
+		scheme:     scheme,
+	}
+}
+
+type defaultRendererFactory struct {
+	logFactory LogFactory
+	scheme     *runtime.Scheme
+}
+
+func (f *defaultRendererFactory) Renderer() treecmd.Renderer {
+	return internalcmd.NewTree(
+		f.scheme,
+		internalcmd.WithLog{
+			Log: f.logFactory.Logger().V(1),
```

Comment:
Is all output of tree debug output? (zaps DebugLevel lvl)

Reply:
I'm fine removing the `v(1)`'s for this and the `builder`. It's a hold over from when we weren't using levels. I've set `zap` to default to error so `Info` level will not be shown unless the user asks for it.

Reply:
The `V(1)`'s are gone now.